docs/fundamentals/flow/monitoring-flow.md

```diff
@@ -0,0 +1,144 @@
+(monitoring-flow)=
+# Monitor Flow
```

Comment:
I think there is a bit of an issue in the layout of this section, apologies if i saw this before and didn't say anything then. But basically, we directly parachute into monitoring only select ports etc., when that is a slightly more advanced feature to me. The simple case would be turning on monitoring for the entire flow, and I would lead with that.

Reply:
ok fair